app/src/common/shared/com/igalia/wolvic/downloads/DownloadsManager.java

```diff
@@ -205,7 +280,7 @@ public List<Download> getDownloads() {
     public boolean isDownloading() {
         return getDownloads().stream()
                 .filter(item ->
-                        item.getStatus() == DownloadManager.STATUS_RUNNING)
+                        item.getStatus() == Download.RUNNING)
```

Comment:
The integer returned by `Download.getStatus()` will be one of the values in `Download.Status`:

```java
public class Download {

    @IntDef(value = { UNAVAILABLE, PENDING, RUNNING, PAUSED, SUCCESSFUL, FAILED})
    @interface Status {}
    ...
    private @Status int mStatus;
    ...
    @Status
    public int getStatus() {
        return mStatus;
    }
```

It worked by chance because `DownloadManager.STATUS_RUNNING` and `Download.RUNNING` both have the same integer value (2).